src/endpoints/assets.js

```diff
@@ -224,7 +224,9 @@ router.post('/download', jsonParser, async (request, response) => {
         // Delete if previous download failed
         if (fs.existsSync(temp_path)) {
             fs.unlink(temp_path, (err) => {
-                if (err) throw err;
+                if (err) {
+                    throw err;
+                }
```

Comment:
Side node: Is it really needed to make code style changes like this? There quite a few in this PR, blowing up the actual changes. Same for empty line adds/deletes.
And while I agree with adding trailing commas on array and the like, this formatting here just comes down to taste. If it's short like this, using one line instead of 3 is totally fine.